src/performance/perfcollect/perfcollect

```diff
@@ -1077,12 +1078,6 @@ ProcessArguments()
 		then
 			events=$value
 			i=$i+1
-		elif [ "-collectsec" == "$arg" ]
```

Comment:
It looks like you inadvertently removed the -collectsec functionality.  Can you please put this back?